root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlSvr.cs

```diff
@@ -114,6 +114,11 @@ public SqlConnection DamSqlConnection
                 // コネクションを戻す
                 return _cnn;
             }
+            set
+            {
+                //set the connection
+                _cnn = value;
+            }
```

Comment:
@yusukemaegawa 

Dam is an class of the propman.
- Therefore, the possibility that the user manipulate it is low.
- Therefore, the possibility that let it occur the problem is low.

And, I think that even if problems occured in this situation, it is self-responsibility.